app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[name.lower()] = Person
```

Comment:
This line is incorrectly assigning the `Person` class to the `people` dictionary. It should assign the instance `self` instead, as in `Person.people[name.lower()] = self`.